src/Symfony/Component/Routing/Router.php

```diff
@@ -329,18 +354,37 @@ public function addExpressionLanguageProvider(ExpressionFunctionProviderInterfac
     }
 
     /**
+     * This method is public because it needs to be callable from a closure in PHP 5.3. It should be converted back to protected in 3.0.
+     * @internal
      * @return GeneratorDumperInterface
      */
-    protected function getGeneratorDumperInstance()
+    public function getGeneratorDumperInstance()
```

Comment:
Well, it's not a BC break according to the rules. So we shouldn't wait until 3.0.

 > **If you extend the class and...**
 > * Override a protected method: No (1)
 > 1: Your code may be broken by changes in the Symfony code. Such changes will however be documented in the UPGRADE file.